docker/Dockerfile

```diff
@@ -14,7 +14,9 @@ ENV APP_PATH=${IMG_APP_PATH}
 RUN apt-get update
 
 # Install utils
-RUN apt-get install -y ca-certificates git lsb-release mongodb nodejs npm sudo wget
+RUN apt-get install -y ca-certificates git lsb-release mongodb nodejs curl sudo wget
+# Get latest NPM
+RUN curl -L https://www.npmjs.com/install.sh | sh
```

Comment:
We don't need to install nodejs or npm independently, OWT required version nodejs and npm will be installed when installing owt-server dependencies, so the better way to modify the Dockerfile is to put owt-client install after owt-server and remove nodejs and npm installation with apt install

Reply:
It appears that while NVM does install the newer versions of nodejs and npm, for some reason they aren't set when I attempt to use them after installDeps.sh.  They work fine if I call 'nvm v8.15.0'. Any idea why this is, or if this is expected behavior? 

Also, is it expected for any app being created for owt-server that they would install owt-client as well? If so should I add installing owt-client-javascript to our base Docker file so the app creator doesn't need to? Or is the theory that they could want to use owt-client-native instead, so its better to let them decide?

Thanks.

Reply:
nvm is a node version manager, you should run 'nvm use v8.15.0' to specify which version do you use.  OWT install will also install nvm and set related command to specify which node version to use, so put jssdk build after owt-server is better since in this Dockerfile it just provides all dependencies for packing OWT.

owt-client-javascript contains js sdk and sample app, if users are developing js app, they still need the js sdk to communicate with owt-server, what they do is to replace the sample app with their app, so adding owt-client-javascript in the base Dockerfile is necessary.

Reply:
Thanks, I'm going to close this PR and make another that uses nvm to fix our issue instead.